maintainers/maintainer-list.nix

```diff
@@ -1658,6 +1658,11 @@
     github = "fare";
     name = "Francois-Rene Rideau";
   };
+  farlion = {
+    email = "florian.peter@gmx.at";
+    github = "workflow";
+    name = "Florian Peter";
+  };
```

Comment:
Separate this into a different commit, same PR

Reply:
Done, thanks